lib/inspec/config.rb

```diff
@@ -360,6 +360,12 @@ def validate_reporters!(reporters)
       end
 
       raise ArgumentError, "The option --reporter can only have a single report outputting to stdout." if stdout_reporters > 1
+
+      if (truncation = @merged_options["reporter_message_truncation"])
+        unless truncation == "ALL" || truncation.to_i.to_s == truncation
```

Comment:
This is wrong if the value is already an Integer. Usually it will be a string (via the CLI) but can be an Integer via JSON config.
@clintoncwolfe 's PR for typing inputs will also cause this to be wrong, as that PR causes inputs to be typed where possible

Reply:
#5011 will only impact the `--input` CLI option

Reply:
D'oh, good point!